### ISSUE TITLE
Darkmode color improvements

### ISSUE THREAD

**Issue by AnttiRumpunen** [fmidev/weather-app#1289]
Current full black background + bright white text are actually bad. Let's change to more greyish and softer white - to be defined later...

https://medium.com/@mohitphogat/dark-mode-done-right-and-why-most-apps-get-it-wrong-a75f90aab30a
